cilium/cilium.io
# feat: adding hetzner to adopters page

**This PR adding hetzner card to adopters page:**


**Steps to test**

- Open the [page]()
- Confirm that everything looks and works as expected
